gen/definitions/segment_routing_te.yaml

```diff
@@ -0,0 +1,64 @@
+---
+name: Segment Routing Traffic Engineering
```

Comment:
We should change the name to `Segment Routing TE` to be consistent.

Reply:
done